python/pyspark/sql/column.py

```diff
@@ -326,6 +326,27 @@ def between(self, lowerBound, upperBound):
         """
         return (self >= lowerBound) & (self <= upperBound)
 
+    @since(1.5)
+    def In(self, *values):
+        """
+        A boolean expression that is evaluated to true if the value of this
+        expression is any of the given columns.
+        NOTE: Normally, we shold name this function the small case `in`. However, `in` is
+        a reserved word in Python. So we can't help naming this the upper case `In`.
+
+        >>> df.select(df.name, df.age, df.age.In(2, 4)).show()
+        +-----+---+---------+
+        | name|age|(age = 2)|
+        +-----+---+---------+
+        |Alice|  2|     true|
+        |  Bob|  5|    false|
+        +-----+---+---------+
+        """
+        for v in values:
```

Comment:
This approach will not scale if you have many values, please call the java API  `in`.